src/Microsoft.AspNetCore.Server.IIS/Core/IISHttpContext.cs

```diff
@@ -223,9 +230,9 @@ private async Task ProduceStart(bool flushHeaders)
                 }
                 // Client might be disconnected at this point
                 // don't leak the exception
-                catch (IOException)
```

Comment:
Can an IOException still happen here?

Reply:
I changed AsyncIO to throw ConnectionResetException that is IOException but is a bit more specific.